src/java2yaml/Common/Utility/CopyUtility.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.Content.Build.Java2Yaml
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+
+    public static class CopyUtility
+    {
+        public static int CopyFiles(List<string> sourcePath, string targetPath)
+        {
+            Guard.ArgumentNotNullOrEmpty(targetPath, nameof(targetPath));
+
+            var fileCopyed = 0;
+
+            foreach (var file in sourcePath)
+            {
+                File.Copy(file, Path.Combine(targetPath, Path.GetFileName(file)), overwrite: true);
+                fileCopyed++;
+            }
+
+            return fileCopyed;

```

Comment:
`fileCopyed` will always equal to `sourcePath.Count`. No need to calculate here.